openid-4-verifiable-presentations-1_0.md

```diff
@@ -2052,7 +2052,7 @@ Out of the Authorization Request parameters defined in [@!RFC6749] and (#vp_toke
 * `transaction_data`
 * `dcql_query`
 
-The `client_id` parameter MUST be omitted in unsigned requests defined in (#unsigned_request). The Wallet determines the effective Client Identifier from the Origin. The effective Client Identifier is composed of a synthetic Client Identifier Scheme of `web-origin` and the Origin itself. For example, an Origin of `https://verifier.example.com` would result in an effective Client Identifier of `web-origin:https://verifier.example.com`. The transport of the request and Origin to the Wallet is platform-specific and is out of scope of OpenID4VP over the W3C Digital Credentials API. The Wallet MUST ignore any `client_id` parameter that is present in an unsigned request.
+The `client_id` parameter MUST be omitted in unsigned requests defined in (#unsigned_request). The Wallet MUST ignore any `client_id` parameter that is present in an unsigned request. The security properties that are normally provided by the Client Identifier are achieved by binding the response to the Origin it was received from. The transport of the request and Origin to the Wallet is platform-specific and is out of scope of OpenID4VP over the W3C Digital Credentials API.
```

Comment:
Didn't we agree that client_id is omitted from signed requests too? so `client_id` needs to be removed from line 2045 too

```suggestion
The Wallet MUST ignore any `client_id` parameter that is present in either signed and unsigned requests. The security properties that are normally provided by the Client Identifier are achieved by binding the response to the Origin it was received from. The transport of the request and Origin to the Wallet is platform-specific and is out of scope of OpenID4VP over the W3C Digital Credentials API.
```

Reply:
client_id is omitted for the response, not the request for signed requests

Reply:
Agree with Christian, we didn't agree to omit from signed requests, and client_id in signed requests is required to know how to verify the signature (it tells you how to find the key etc) so we can't omit it.

Reply:
exactly. We don't want to change the signed request payload (as it governs the handling of the client id scheme specific credential). We drop the idea to use the client id as audience in the credential presentation and use the origin instead.

Reply:
ah ok, got it - thanks for clarifying